<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1211</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgWidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0.00568182, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(255, 255, 255, 255))}</string>
    </property>
    <widget class="QLabel" name="mainTitle">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>751</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);
font: 24pt &quot;Palatino Linotype&quot;;
padding:10px;
border-radius: 28px;
</string>
     </property>
     <property name="text">
      <string>Welcome to the Device Control Module</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 0, 0);
	font: 14pt &quot;MS Sans Serif&quot;;
	color: rgb(255, 255, 255);
	border-radius: 40px;
}

QPushButton:hover {
	background-color: rgb(255, 60, 60);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../assets/exit.png</normaloff>../../assets/exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="connectedStatusIcon">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>25</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>../../assets/connected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="connectedStatusText">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 170, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="text">
      <string>CONNECTED</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="editAOO_Button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>150</width>
       <height>90</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;
	
	font: 14pt &quot;MS Reference Sans Serif&quot;;
}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Edit AOO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editVOO_Button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>150</width>
       <height>90</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;

	font: 14pt &quot;MS Reference Sans Serif&quot;;

}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Edit VOO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editAAI_Button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>430</y>
       <width>150</width>
       <height>90</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;

	font: 14pt &quot;MS Reference Sans Serif&quot;;

}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Edit AAI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editVVI_Button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>540</y>
       <width>150</width>
       <height>90</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;

	font: 14pt &quot;MS Reference Sans Serif&quot;;

}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Edit VVI</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentParameters">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>170</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Current Parameters</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lowerLimit">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>210</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lower Rate Limit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="upperLimit">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Upper Rate Limit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lowerLimit_Value">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="upperLimit_Value">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>230</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARP">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ARP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="VRP">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>280</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>VRP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ARP_Value">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="VRP_Value">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>280</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="VRP_2">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Atrial Amplitude:</string>
     </property>
    </widget>
    <widget class="QLabel" name="VRP_3">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>350</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ventricular Amplitude:</string>
     </property>
    </widget>
    <widget class="QLabel" name="VAmp_Value">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>350</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="AAmp_Value">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>330</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="VPW_Value">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>420</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="VRP_4">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>400</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Atrial PW:</string>
     </property>
    </widget>
    <widget class="QLabel" name="VRP_5">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ventricular PW:</string>
     </property>
    </widget>
    <widget class="QLabel" name="APW_Value">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>400</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QWidget" name="ParamWidget" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>210</y>
       <width>351</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#ParamWidget{border: 5px solid black;}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Current Device UID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="device_Value">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0xF00D</string>
      </property>
     </widget>
     <widget class="QLabel" name="loggedInText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Logged In As:</string>
      </property>
     </widget>
     <widget class="QLabel" name="user_Value">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
